extractor/cli/go-autobuilder/go-autobuilder.go

```diff
@@ -508,8 +508,13 @@ func main() {
 	}
 
 	var cmd *exec.Cmd
-	log.Printf("Running extractor command '%s %s ./...' from directory '%s'.\n", extractor, modMode, cwd)
-	cmd = exec.Command(extractor, modMode.String(), "./...")
+	if depMode == GoGetWithModules && modMode.String() != "" {
```

Comment:
```suggestion
	if depMode == GoGetWithModules && modMode != ModUnset {
```

That said, `modMode` can be unset by `modMode = modModIfAvailable`, so it's probably worth having a separate thing to track that.

Or, actually, we could move the logic for versions into `ModMode.String()` and then it would make everything else a bit nicer.

Reply:
Cleaned this up